schemas/dataset-schema.json

```diff
@@ -442,7 +470,7 @@
                 "display": { "$ref": "definitions.json#/display" },
                 "presentation_license": {
                   "type": "object",
-                  "description": "Custom license for the indicator. This is usally not needed and the value from `license` is used.",
+                  "description": "License to display for the indicator, overriding `license`.",
```

Comment:
I am not sure what's the best way to write the field `license`. It would be something like `tables[].variables[].license`? Either write it as that or `indicator.license`?

Reply:
I didn't even know about this field called "presentation_license". Clearly it should be under "presentation". Everything around licenses is still unclear. We may have to rephrase these things. Let's leave it for not to not block other things.